src/fides/api/schemas/connection_configuration/connection_secrets_bigquery.py

```diff
@@ -15,14 +15,16 @@ class KeyfileCreds(BaseModel):
 
     type: Optional[str] = None
     project_id: str = Field(title="Project ID")
-    private_key_id: Optional[str] = Field(None, title="Private Key ID")
-    private_key: Optional[str] = Field(None, sensitive=True)
+    private_key_id: Optional[str] = Field(default=None, title="Private Key ID")
+    private_key: Optional[str] = Field(
+        default=None, json_schema_extra={"sensitive": True}
+    )
     client_email: Optional[EmailStr] = None
-    client_id: Optional[str] = Field(None, title="Client ID")
-    auth_uri: Optional[str] = Field(None, title="Auth URI")
-    token_uri: Optional[str] = Field(None, title="Token URI")
+    client_id: Optional[str] = Field(default=None, title="Client ID")
```

Comment:
Why do we need to add the `default` keyword here? Is this something we should change for all usages of `Field`?

Reply:
we don't have to add it, it's not required but doesn't hurt -

Reply:
Got it! Thanks